File: AlertToCareAPI/Models/BedData.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AlertToCareAPI.Models
{
    public class BedData
    {
        [Key]
        public string BedID { get; set; }

        [ForeignKey("BedConfigurationID")]
        public int BedConfigurationID { get; set; }
        public string Department { get; set; }
        public string OccupancyStatus { get; set; }

    }
}

File: alert-care/src/app/models/bed-data.ts
export class BedData {
    bedID: string;
    bedConfigurationID: number;
    department: string;
    occupancyStatus: string;

    constructor(bedId:string, bedConfigurationID:number, dept:string, status:string){

        this.bedID = bedId;
        this.bedConfigurationID = bedConfigurationID;
        this.department = dept;
        this.occupancyStatus = status;
    }
}
